pychromecast/controllers/media.py

```diff
@@ -67,6 +69,15 @@ def __init__(self):
         self.media_metadata = {}
         self.subtitle_tracks = {}
         self.current_subtitle_tracks = []
+        self.last_updated = time.time()
+
+    def get_current_time(self):
+        """ Get current seek time of video """
+        if self.player_state == MEDIA_PLAYER_STATE_PLAYING:
+            current_time = self.current_time + (time.time()-self.last_updated)
+        else:
+            current_time = self.current_time
+        return current_time
```

Comment:
I personally would have made this `return` statement inline, it's a bit more readable:

```python
if self.player_state == MEDIA_PLAYER_STATE_PLAYING:
    # Add time since last update
    return self.current_time + (time.time()-self.last_updated)
# Not playing, return last reported seek time
return self.current_time
```